.changeset/curvy-falcons-count.md

```diff
@@ -0,0 +1,17 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Adds the option to only run image optimization on images during build-time. :warning: This mode does not work with on-demand (SSR) image optimization.
```

Comment:
You are right, it would not render a `warning` section, but I hope it would render an emoji (https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md#warning)

However since that is also unclear, I'll just replace this with bold text saying `Warning:`.